### PR TITLE
Override default moment relative time - Tamil, Kyrgyz, Marathi

### LINKED ISSUE #2152: Override moment relative timestamp translations - Tamil, Kyrgyz, Marathi
**Is your feature request related to a problem? Please describe.**
Apply the correct wording for relative timestamp to match what we have currently on PAL.

**Describe the solution you'd like**
Override moment's default relative time in `psammead-locales` to have the correct translations. 
**Services**
- Tamil
- Kyrgyz
- Marathi

**Example**
```js
moment.updateLocale('yo', {
  relativeTime: {
    past: '%s sẹ́yìn',
    s: 'ìṣẹ́jú aayá die',
    m: 'ìṣẹ́jú kan',
    mm: 'ìṣẹ́jú %d',
    h: 'wákàtí kan',
    hh: 'wákàtí %d',
  },
});
```
More details on the spreadsheet. 

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #2152

**Overall change:** Override default moment relative time so that we can have the correct localised relative time. 

**Code changes:**

- Add `ky` locale for Kyrgyz
- Override moment's relative time for `ta` (Tamil) and `mr` (Marathi)

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/moment/mr.js [hunk 1]
```diff
@@ -3,4 +3,11 @@ require('moment/locale/mr');
 
 moment.updateLocale('mr', {
   postformat: null,
+  relativeTime: {
+    past: '%sपूर्वी',
```
Comment (DenisHdz) [inline packages/utilities/psammead-locales/moment/mr.js]: I believe there should be a space.
```suggestion
    past: '%s पूर्वी',
```

**Review by DenisHdz — APPROVED** [review]
Looks good 👍

**Review by dr3 — APPROVED** [review]

**Review by DenisHdz — DISMISSED** [review]
~~This PR needs to also update the months.~~

**Comment by staylos92** [comment]
LGTM 👍

**Review by DenisHdz — APPROVED** [review]

**Pull request merged**